backend/src/events/controllers.ts

```diff
@@ -279,6 +279,7 @@ const getEvent = async (eventID: string) => {
       id: eventID,
     },
     include: {
+      attendees: true,
```

Comment:
this is tricky. The manage attendees page does a request of the same type (/event/:eventid) to grab the details for a specific event. But because this flag is set - it returns the attendees with it...which is not needed. The attendees list can grow really large. The screenshot below shows 60.

<img width="414" alt="Screenshot 2024-06-19 at 22 17 47" src="https://github.com/cornellh4i/lagos-volunteers/assets/59776300/4b93f142-ac45-429f-8670-02496b48cccb">

Reply:
I agree. I needed the full attendees object for a couple of reasons but I thought about it and realized I could just make new API endpoints that do the same thing. Fixed it on my other PR and I'll merge the changes over here once that's merged.